src/m_menu.c

```diff
@@ -3409,7 +3410,8 @@ setup_menu_t stat_settings2[] =
 
   {"PREFER CRISPY HUD OVER BOOM HUD"  ,S_YESNO ,m_null,M_X,M_Y+stat2_crispyhud*M_SPC, {"crispy_hud"}},
   {"USE STANDARD DOOM FONT FOR WIDGETS", S_CHOICE,m_null,M_X,M_Y+stat2_hudfont*M_SPC, {"hud_widget_font"}, 0, NULL, show_widgets_strings},
-  {"DRAW BAR GRAPHS IN BOOM HUD", S_YESNO,m_null,M_X,M_Y+stat2_bargraphs*M_SPC, {"hud_draw_bargraphs"}},
+  {"WIDESCREEN WIDGET ARRANGEMENT", S_YESNO,m_null,M_X,M_Y+stat2_widescreen*M_SPC, {"hud_widescreen_widgets"}},
```

Comment:
Or "alignment"? Or any better idea anybody?

Reply:
"widescreen-aware widgets" ?

Reply:
Hm, I'm not sure I like that better. I think I'll merge it as is, we can discuss the precise wording later.